ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-multi-stable-4.12.yaml

```diff
@@ -0,0 +1,155 @@
+base_images:
```

Comment:
The file rename is unexpected.

Reply:
Hmm, I renamed it as the file will only hold the multi-arch to multi-arch upgrade. Any side effect if it's renamed?

Reply:
Originally, it contains only multi-arch upgrade test in it, with the same name style as the others.

Reply:
Also i think we do not support cross arch upgrade at the moment, e.g, `amd64-nightly-4.13-upgrade-from-arm64-stable-4.12.yaml`, so there is no need to duplicate the arch info twice in the file name (thus in the job name, which makes the job name longer).

Reply:
Makes sense. Thanks!